hue/hue/base/hue-mysql-secret.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hue-mysql-secret
+  annotations:
+    template.openshift.io/expose-username: "{.data['database-user']}"
```

Comment:
How are these used?

Reply:
No idea. This comes from the original implementation in the legacy ODH operator.

Reply:
removed